policy/lib/sbom/sbom.rego

```diff
@@ -10,16 +10,32 @@ import rego.v1
 # image's filesystem. This fallback exists for legacy purposes and support for it will be removed
 # soon.
 
-all_sboms := array.concat(cyclonedx_sboms, spdx_sboms)
+# try fetching SBOMs from image attachments first, i.e. the ones pointed to by the SBOM_BLOB_URL
+all_sboms := sboms if {
+	sboms := array.concat(cyclonedx_sboms, spdx_sboms)
+	count(sboms) > 0
+}
+
+# fallback to reading the image files from input or extracting the files from
+# the image under test
+else := array.concat(cyclonedx_sboms_from_image, spdx_sboms_from_image)
```

Comment:
I didn't know you have an `else` statement with a new line in-between. Weird, but cool :sweat_smile: